frameit/lib/frameit/trim_box.rb

```diff
@@ -0,0 +1,33 @@
+module Frameit
+  # Represents the MiniMagick trim bounding box for cropping a text image
+  class Trimbox
+    attr_accessor :width # width of the trim box
+    attr_accessor :height # height of the trim box
+    attr_accessor :offset_x # horizontal offset from the canvas to the trim box
+    attr_accessor :offset_y # vertical offset from the canvas to the trim box
+
+    # identify_string: A string with syntax "<width>x<height>+<offset_x>+<offset_y>". This is returned by MiniMagick when using function .identify with format("%@"). It is also required for the MiniMagick .crop function.
+    def initialize(identify_string)
+      UI.user_error!("Trimbox can not be initialised with an empty 'identify_string'.") unless identify_string.length > 0
```

Comment:
I copied this logic from another Class.
When testing with faulty input, I see the Error in Terminal, but Frameit just continues.
It does not write a framed screenshot.

Is this intended behaviour?

Reply:
It's possible this error is getting rescued somewhere and continuing execution. I would say leave it as is, this class looks 💯 !